mne/io/nirx/nirx.py

```diff
@@ -0,0 +1,204 @@
+# Authors: Robert Luke <mail@robertluke.net>
+#
+# License: BSD (3-clause)
+
+import configparser as cp
```

Comment:
`cp` is easily confused with `copy` so maybe
```suggestion
from configparser import ConfigParser, RawConfigParser
```